data/avmoo

```diff
@@ -1,4 +1,3 @@
-avmoo.casa
+avmoo.click
 avmoo.com
-avmoo.cyou
```

Comment:
The `avmoo.casa` and `avmoo.cyou` are no longer available.

Reply:
good catch!